spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/web/HeaderKeys.java

```diff
@@ -0,0 +1,9 @@
+package org.springframework.cloud.sleuth.instrument.web;
+
+//Annotation key holding class for request and response headers
+abstract class HeaderKeys {
```

Comment:
bikeshed, but I usually go for final class with private ctor for constant holders.